librz/core/cmd_descs/cmd_debug.yaml

```
@@ -1009,3 +1009,36 @@ commands:
             args:
               - name: number
                 type: RZ_CMD_ARG_TYPE_NUM
+  - name: dx
+    summary: Code injection commands
+    subcommands:
+      - name: dx
+        summary: Inject opcodes
+        cname: cmd_debug_inject_opcode
+        args:
+          - name: opcode
+            type: RZ_CMD_ARG_TYPE_STRING
+      - name: dxa
+        summary: Assemble code and inject
+        cname: cmd_debug_inject_assembly
+        args:
+          - name: asm
+            type: RZ_CMD_ARG_TYPE_STRING
+      - name: dxe
+        summary: Compile RzEgg expression and inject
+        cname: cmd_debug_inject_egg
+        args:
+          - name: egg
```

```suggestion
          - name: expression
```
how about expression?